hips/tiles/surveys.py

```diff
@@ -127,7 +132,12 @@ def tile_format(self) -> str:
     @property
     def base_url(self) -> str:
```

Comment:
Do we have some test cases for the different cases? (base URL retrieved from hips_service_url, from moc_access_url or directly from the properties URL)